docs-src/concepts/what-is-the-temporal-web-ui.md

```diff
@@ -148,3 +148,7 @@ For details, see [Securing your data](/production-readiness/develop#securing-you
 The UI supports both a [Codec Server endpoint](/production-readiness/develop#web-ui) and the `tctl` plugin port.
 
 For details on setting the Codec Server endpoint, see [Codec Server setup](/production-readiness/develop#codec-server-setup).
+
+### Labs Mode
+
+The Web UI provides a Labs mode for users to try out upcoming, production-ready UI features and improvements. When off, users will experience the current UI. Features will move in and out of Labs mode according to demand and feedback. Labs mode can be turned on or off at any time in the left navigation bar via the Labs icon.
```

Comment:
Is it "Labs Mode", "Labs mode", or "labs mode" ?
That dictates which letter gets capitalized depending on where it is used.

My guess is it is "labs mode" generically - 
If so, I updated the suggestion below to match.

```suggestion
### Labs mode

The Web UI provides a "labs mode" for users to try out upcoming, production-ready UI features and improvements.
When off, users will experience the current UI.
Features will move in and out of labs mode according to demand and feedback.
Labs mode can be turned on or off at any time in the left navigation bar via the Labs icon.
```
moving each sentence to new line as well

Reply:
It's generic, so yup made it Labs mode. Only thing I changed from your suggestion is "labs" mode instead of "labs mode"